src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -380,7 +381,7 @@ FixedwingAttitudeControl::vehicle_status_poll()
 
 				int32_t vt_type = -1;
 
-				if (param_get(param_find("VT_TYPE"), &vt_type) == PX4_OK) {
+				if (param_get(param_handle(px4::params::VT_TYPE), &vt_type) == PX4_OK) {
```

Comment:
VTOL needs to be optional for FW & MC controllers.

Reply:
there are many issues about mixing VTOL/Rover/Multicopter/Fixwing